lang/swi-prolog-devel/Portfile

```diff
@@ -32,9 +32,9 @@ master_sites        https://www.swi-prolog.org/download/devel/src/
 distname            swipl-${version}
 dist_subdir         swi-prolog
 
-checksums           rmd160  3a755d24b3794cf3eac080223650174d7957c121 \
-                    sha256  4718c426c58d09e76bd84c109ae59f47b8cc261213a47796296b3733f455a195 \
-                    size    11797898
+checksums           rmd160  8e501ba91c330b289d2c4a326f24282da5bbec42 \
+                    sha256  66cbe9652528c831d26fa9999f3a088ab5b9126115d33f48442ac0e52d4d2be2 \
+                    size    11818020
 
 # to avoid the following problem: fatal error: 'threads.h' file not found
 compiler.blacklist-append {clang < 900}
```

Comment:
@JanWielemaker I cannot remember the outcome of the discussion about this anymore.... is the blacklisting still needed here (I think you mentioned checks in the CMake files that avoids the issue). If it is indeed not needed anymore would you mind opening a PR to remove these two lines and the `PortGroup           compiler_blacklist_versions 1.0` include?